tests/test_security_hardening.py

```
@@ -116,6 +128,38 @@ def test_admin_and_inference_tokens_are_separate() -> None:
     assert "trace" not in inference_body["orchestration"]
 
 
+def test_inference_token_cannot_read_admin_get_surfaces() -> None:
+    server = build_server(
+        build(),
+        port=0,
+        security=SecurityConfig(auth_token="", admin_token="admin_secret", inference_token="inference_secret"),
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "admin_token"

(S106)